packages/grafana-toolkit/src/cli/templates/component.tsx.template.ts

```diff
@@ -1,8 +1,8 @@
-export const componentTpl = `import React, { FC } from 'react';
+export const componentTpl = `import React from 'react';
 
 export interface Props {};
 
-export const <%= name %>: FC<Props> = (props) => {
+export const <%= name %> = (props: Props) => {
```

Comment:
@mckn -- should I pull out the toolkit changes?  or does it seem OK as part of this PR?

Reply:
I think it's fine to keep it. 👍 
FYI - we've [deprecated the command that uses them](https://github.com/grafana/grafana/blob/main/packages/grafana-toolkit/src/cli/index.ts#L87-L102) with the intention to remove it (and these templates) completely in Grafana 10 in favour of [react-gen-component](https://www.npmjs.com/package/react-gen-component).